### PR TITLE
add vhost in from of requests in access_log

### PR DESCRIPTION
**Pull request by mbaechler**
this needs real-life testing, can I let you try on your ideasbox ?

### CODE REVIEW

File: extras/nginx/ideascube [hunk 1]
```diff
@@ -3,6 +3,13 @@ upstream ideascube {
     server unix:///tmp/ideascube.sock;
 }
 
+#add host in from of request to track server_name in logs
```
Comment (bochecha) [inline extras/nginx/ideascube]: I don't understand this comment.

File: extras/nginx/ideascube [hunk 2]
```diff
@@ -14,6 +21,8 @@ server {
     server_name ideasbox.lan www.ideasbox.lan ideasbox.local www.ideasbox.local koombook.lan www.koombook.lan koombook.local www.koombook.local bibliotecamovil.lan www.bibliotecamovil.lan;
     charset     utf-8;
 
+    
+
```
Comment (bochecha) [inline extras/nginx/ideascube]: Why those new lines?

Comment (mbaechler) [inline extras/nginx/ideascube]: it's noise, I first expected to put configuration there and forgot to remove them.

File: extras/nginx/ideascube (hunk 1 above)
Comment (bochecha) [inline extras/nginx/ideascube]: Ah, that makes more sense now, thanks.

Can you merge both commits?

**Comment by letompouce** [comment]
Thinking a bit more about it, we want this log format for all logs, so it should be configured right into the main nginx configuration file and not only for this vhost. Somewhere in https://github.com/ideascube/ansiblecube/blob/oneUpdateFile/roles/nginx/files/nginx.conf

**Comment by mbaechler** [comment]
of course, feel free to close this PR and do the modification wherever it best fits.

**Comment by Toniob** [comment]
I think you should use "$host" instead of "$server_name" in the log format

**Comment by letompouce** [comment]
Fixed by https://github.com/ideascube/ansiblecube/commit/9b046ee10490fa9696427a74018dc75a813f9cbc

**Pull request closed without merging**